Samples/Notifications/Push/README.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "Push Notifications Sample" 
+page_type: sample
+languages:
+- cpp
+products: 
+- windows-app-sdk
+description: "Demonstrates how to use the windows app SDK Push Notifications APIs from an unpackaged app"
+urlFragment: PushNotifications
+---
+# Push Notifications Sample
+This sample demonstrates:
+1. How an app can use the Windows App SDK to request a WNS Channel URI
+    * The WNS Channel URI uniquely identifies the app for a current user on a specific device, allowing other apps and servers to send notifications to this app.
+2.	How to use the Windows App SDK to register an app for foreground and background activation and handle the activations.
+## Requirements
+* A web API tool (like Postman or fiddler) to send notifications.
+   * The Windows App SDK push notifications APIs provides everything an app needs register for and receive notifications but it's expected that an external server app will be sending the notifications. The web API tool is used to emulate that external server app to send notifications.
+* An AAD Remote Identifier for your app.
+   * To obtain an AAD Remote identifier, follow these [instructions](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory).
```

Comment:
REmove en-us from the links. All web links need to be locale affine.

Reply:
Is there a reason why we say "AAD Remote Identifier" instead of "Azure AppId", since that's what it is called in the quickstart? Also, my comment on concern over having the full URL is also applicable here.

Reply:
no reason, I've replaced all AAD usages I could find.